zoo-css-challenge/style.css

```diff
@@ -192,4 +237,9 @@ footer {
 
 .programs .card {
   display: block;
+  flex-wrap: wrap;
+  margin: 1px 0 0 1px;
 }
```

Comment:
You can remove this block of code after adding the card-container class. See previous comment to see how to arrange the cards in the programs section side by side.